src/core/installAddon.js

```diff
@@ -45,12 +47,16 @@ export function makeProgressHandler(dispatch, guid) {
   };
 }
 
-export function mapStateToProps(state, ownProps, { _tracking = tracking } = {}) {
-  return {
-    installTheme(node, guid, name, _themeAction = themeAction) {
-      _themeAction(node, THEME_INSTALL);
-      _tracking.sendEvent({ action: 'theme', category: INSTALL_CATEGORY, label: name });
-    },
+export function makeMapStateToProps({ WrappedComponent }) {
+  return function mapStateToProps(state, ownProps, { _tracking = tracking } = {}) {
+    return {
+      WrappedComponent,
+      hasAddonManager: typeof window !== 'undefined' && addonManager.hasAddonManager(),
```

Comment:
both `hasAddonManager` and `installTheme` don't appear to use `state` in any way. Do they need to be in `mapStateToProps`? Could they just be added directly to props in `withInstallHelpers` ? The `mapStateToProps` function is a bit heavy since it's executed on every state change.